students/KyleBarry/lesson03/test_customer.py

```diff
@@ -56,7 +72,7 @@ def test_list_active_customers():
 
     result = list_active_customers()
     logging.info(result)
-    assert result == 1
+    assert result == 8
```

Comment:
Your tests are dependent on order, which is difficult to maintain, and means it is impossible to run tests in isolation. Consider using fixtures. https://docs.pytest.org/en/latest/fixture.html